.authors.yml

```diff
@@ -8,7 +8,7 @@
   aliases:
   - Mike Sarahan
   - Michael Sarahan
-  num_commits: 98
+  num_commits: 298
```

Comment:
??

Reply:
Unfortuantely, this is a "feature" of rever, we can either have a correct `.mailmap` or a correct commit count in `.authors.yml`. We value a correct mailmap more for author deduplication so nothing to see here. The correct commit count is always available via `git shortlog -se`.